JASP-Desktop/analysis/options/optionstable.cpp

```diff
@@ -45,8 +45,11 @@ Json::Value OptionsTable::asJSON() const
 
 void OptionsTable::deleteOldValues()
 {
-	for(Options *row : _value)
-		delete row;
+	auto i = std::begin(_value);
+
+	while(i != std::end(_value)) {
+		_value.erase(i);
```

Comment:
Why would you do this? 

`erase` does not delete a pointer... It simply removes the pointer from the list.
And we do clear afterwards? So this is pointless, the idea here was to avoid a memory leak caused by throwing away pointers to objects without deleting/destroying them..